sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_client.py

```diff
@@ -95,7 +115,28 @@ def wait_until_done(self, job_id, **kwargs):
         :return: JobStatusDetail
         :rtype: JobStatusDetail
         """
-        pass  # pylint: disable=unnecessary-pass
+
+        pipeline_response = self.get_job_status(
```

Comment:
just realized, now that we've wrapped the `get_job_status` method to return the exposed model it won't work here. We should call the equivalent generated code method instead:
```suggestion
        pipeline_response = self._client.document_translation.get_operation_status(
```